bin/ember-template-lint.js

```diff
@@ -331,11 +340,17 @@ async function run() {
     }
 
     if (options.updateTodo) {
-      await linter.updateTodo(
-        linterOptions,
-        fileResults,
-        getTodoConfig(options.workingDirectory, getTodoConfigFromCommandLineOptions(options))
+      let todoConfig = getTodoConfig(
+        options.workingDirectory,
+        getTodoConfigFromCommandLineOptions(options)
       );
+
+      let [added] = await linter.updateTodo(linterOptions, fileResults, todoConfig);
```

Comment:
We should probably also keep track of removals, right? In theory someone could use `--update-todo` instead of `--fix` to clear a prior todo that is no longer valid.

Reply:
Yep. I wasn't sure how far we wanted to go in reporting this stuff. I'm happy to add it.

Reply:
Done.